.jenkins/pytorch/common.sh

```diff
@@ -157,6 +157,28 @@ if [[ "$BUILD_ENVIRONMENT" == *pytorch-xla-linux-bionic* ]] || \
     conda install -q -y cmake
   fi
 fi
+if which conda; then
+  # MKL is provided via conda. Usually, users will activate their conda
+  # environment before building a project, which will add LDFLAGS
+  # (and much more). Without LDFLAGS, we get
+  # "not found (try using -rpath or -rpath-link)"
+  # errors when looking for MKL libraries while linking
+  # Without the ldconfig conda-provided libs are not found at runtime
+  if [[ "$BUILD_ENVIRONMENT" != *pytorch-win-* ]]; then
+    CONDA_LIBS=${CONDA_PREFIX:-"$(dirname $(dirname $(which conda)))/lib"}
+    export LDFLAGS="-Wl,-rpath-link=${CONDA_LIBS}"
+    # TODO: once the new docker images in this PR (PR 37737) are created, this is
+    # redundant (it should be part of .circleci/docker/common/install_conda.sh)
+    if [ -e /etc/ld.so.conf.d/conda-python.conf ]; then
+      echo Now safe to remove this temporary fix from .jenkins/pytorch/commmon.sh
```

Comment:
Will someone monitor logs for this message? Otherwise, I doubt it will draw any attention.

Reply:
Yes, I will. It will require another PR to use the newly tagged images, which will then enable removing this message.